styles.js

```diff
@@ -1,14 +1,229 @@
 import { createGlobalStyle } from "styled-components";
 
 export default createGlobalStyle`
-  *,
-  *::before,
-  *::after {
+:root {
+  --color-background: #f0f5f1; 
+  --color-text-primary: #2e4d36; 
+
+  --color-background-cards: #ffffff; 
+
+  --color-button-text: #ffffff; 
+  --color-button-border: #b5cbb7; 
+
+  --color-button-background: #5b8a72; 
+  --color-button-hover-background: #4a725c; 
+
+  --color-button-save: #4caf50;
+  --color-button-save-hover: #3e8e41; 
+
+  --color-button-cancel: #b58362; 
+  --color-button-cancel-hover: #a17450; 
+
+  --color-button-delete: #e53935; 
+  --color-button-delete-hover: #c62828;
+
+  --color-button-favourite: #d2a85d; 
+  --color-button-favourite-hover: #b5884b; 
+
+  --color-button-add: #4a725c;
+  --color-button-add-hover: #3c5b4a; 
+
+  --color-button-edit: #c07a2f;
+  --color-button-edit-hover: #b5884b; 
+
+  --color-link-see-more: #6c8f5c;
+  --color-link-see-more-hover: #597547;
+
+  --color-button-filter: #4a725c; 
+  --color-button-filter-tag: #3c5b4a; 
+  --color-button-filter-hover: #3c5b4a; 
+
+  --color-label: #555;
+
```

Comment:
same comment as earlier, if you use the same hex for multiple colours, maybe it is easier to group them. 
so instead of` --color-button-add: #4a725c; ` and `--color-button-filter: #4a725c` you would have: --light-green: #4a725c